docs/Backwards_Compatibility.md

```diff
@@ -122,6 +122,105 @@ not subject to the backwards compatibility design:
 should be considered part of our API and subject to backwards compatibility
 requirements.
 
+## Feature/API Stage Requirements
+
+Gatekeeper follows the Kubernetes [API versioning conventions](https://kubernetes.io/docs/reference/using-api/#api-versioning) 
+and maturity levels (Alpha, Beta, GA) to signal stability and compatibility guarantees. 
+
+### Alpha Stage Requirements
+
+Alpha features and APIs are experimental and may change or be removed without notice. 
+They are disabled by default and require explicit opt-in.
+
+**Requirements:**
+- Feature must be behind a feature flag or opt-in configuration flag
```

Comment:
This is technically not true for few APIs. 

e.g https://open-policy-agent.github.io/gatekeeper/website/docs/sync/

e.g. Gatekeeper's Config CRD is alpha (v1alpha1), but it does NOT require a feature flag.

I recommend, document “with few exceptions. Starting from Gatekeeper version 3.x.0, this requirement will be enforced.”

Reply:
@ritazh the intent was to indicate that we only need feature flags for features and not the APIS, for example `enable-violation-export` or `operation=generate` vs APIS. I updated the doc to make it a little bit more clear, let me know if its still confusing. Added "startgin from gk.." line as well.